core/src/main/java/org/lflang/federated/generator/FederateInstance.java

```diff
@@ -97,28 +92,20 @@ public FederateInstance(
     this.messageReporter = messageReporter;
     this.targetConfig = targetConfig;
 
-    if (instantiation != null) {
-      // If the instantiation is in a bank, then we have to append
-      // the bank index to the name.
-      if (instantiation.getWidthSpec() != null) {
-        this.name = "federate__" + instantiation.getName() + "__" + bankIndex;
-      } else {
-        this.name = "federate__" + instantiation.getName();
-      }
+    // If the instantiation is in a bank, then we have to append
+    // the bank index to the name.
+    if (instantiation.getWidthSpec() != null) {
+      this.name = "federate__" + instantiation.getName() + "__" + bankIndex;
+    } else {
+      this.name = "federate__" + instantiation.getName();
     }
   }
 
-  /////////////////////////////////////////////
-  //// Public Fields
-
   /**
    * The position within a bank of reactors for this federate. This is 0 if the instantiation is not
    * a bank of reactors.
    */
-  public int bankIndex = 0;
-
-  /** A list of outputs that can be triggered directly or indirectly by physical actions. */
-  public Set<Expression> outputsConnectedToPhysicalActions = new LinkedHashSet<>();
+  public int bankIndex;
```

Comment:
Why is the initialization of bankIndex to 0 removed?

Reply:
Because it's unnecessary. The default initialization is zero.